apps/src/lib/script-editor/ScriptEditor.jsx

```diff
@@ -65,6 +66,13 @@ const ScriptEditor = React.createClass({
         e.preventDefault();
       }
     }
+
+    if (this.loginRequired.checked && this.age13Plus.checked) {
+      if (!confirm('Turning on "age 13+ required" will have no affect as long as '+
+          '"login required" is still on. Are you sure you want to continue?')) {
```

Comment:
For some reason I thought App Lab was age 13+ only for signed in users too, in which case having these both checked, while redundant, is not really a conflict.  It might be more important to warn levelbuilders about a script containing App Lab levels that was _not_ limited to age 13+... or maybe we should just automate that?

Reply:
I think that is the case.
 
I actually added this because while testing I had checked the box, and then became confused as to why it had no effect (because login was still required).

It might be that I should make this setting clearer by calling it something like "13+ required for signed out users"